src/Airtable.php

```diff
@@ -48,9 +48,9 @@ public function table($table)
         return $this;
     }
 
-    public function where($column, $value)
+    public function where($column, $value, $operator = '=')
```

Comment:
Could you adjust this to allow
Airtable::where('id', '>', 5)->get();
Airtable::where('id', 5)->get(); // shorthand for id = 5

this would match Laravel's query builder syntax https://laravel.com/docs/7.x/queries#where-clauses

Reply:
Done!